definitions/npm/axios_v0.15.x/flow_v0.28.x-/axios_v0.15.x.js

```diff
@@ -0,0 +1,117 @@
+declare module 'axios' {
+  declare interface ProxyConfig {
+    host: string;
+    port: number;
+  }
+  declare interface Cancel {
+    constructor(message?: string): Cancel;
+    message: string;
+  }
+  declare interface Canceler {
+    (message?: string): void;
+  }
+  declare interface CancelTokenSource {
+    token: CancelToken;
+    cancel: Canceler;
+  }
+  declare interface CancelToken {
+    constructor(executor: (cancel: Canceler) => void): CancelToken;
+    static source(): CancelTokenSource;
+    promise: Promise<Cancel>;
+    reason?: Cancel;
+    throwIfRequested(): void;
+  }
+  declare interface AxiosXHRConfigBase<T> {
+    adapter?: <T>(config: AxiosXHRConfig<T>) => Promise<AxiosXHR<T>>;
+    auth?: {
+      username: string,
+      password: string
+    };
+    baseURL?: string,
+    cancelToken?: CancelToken;
+    headers?: Object;
+    httpAgent?: mixed; // Missing the type in the core flow node libdef
+    httpsAgent?: mixed; // Missing the type in the core flow node libdef
+    maxContentLength?: number;
+    maxRedirects?: 5,
+    params?: Object;
+    paramsSerializer?: (params: Object) => string;
+    progress?: (progressEvent: Event) => void | mixed;
+    proxy?: ProxyConfig;
+    responseType?: 'arraybuffer' | 'blob' | 'document' | 'json' | 'text' | 'stream';
+    timeout?: number;
+    transformRequest?: Array<<U>(data: T) => U|Array<<U>(data: T) => U>>;
+    transformResponse?: Array<<U>(data: T) => U>;
+    validateStatus?: (status: number) => boolean,
+    withCredentials?: boolean;
+    xsrfCookieName?: string;
+    xsrfHeaderName?: string;
+  }
+  declare type $AxiosXHRConfigBase<T> = AxiosXHRConfigBase<T>;
+  declare interface AxiosXHRConfig<T> extends AxiosXHRConfigBase<T> {
+    data?: T;
+    method?: string;
+    url: string;
+  }
+  declare type $AxiosXHRConfig<T> = AxiosXHRConfig<T>;
+  declare class AxiosXHR<T> {
+    config: AxiosXHRConfig<T>;
+    data: T;
+    headers: Object;
+    status: number;
+    statusText: string,
+    request: http$ClientRequest | XMLHttpRequest
+  }
+  declare type $AxiosXHR<T> = $AxiosXHR<T>;
```

Comment:
There are a few other minor changes like `defaults` & missing `common` in `headers`, etc. I will will send a PR with with everything.